.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
+cache:
+  directories:
+    - node_modules
+
+notifications:
+  email: false
+
 node_js:
-  - '4'
+  - '6'
```

Comment:
Let's give it a try - this was the recommended version from the semantic release cli.